### ISSUE TITLE
Turning pages while swipes is off

### ISSUE THREAD

**Issue by dvcrn** [ddvk/remarkable-hacks#328]
I noticed I kept accidentally hitting some of the swipes so I turned them off, but with them turned off there is also no real way to turn a page from what I can tell. 

When "ExtraUI" is on, there is the <pages> of <pages> section in the sidebar, but the arrows under that are more for jumping between previous/next pages in the visit history instead of going the next/prev page in the notebook. 

Is it possible to turn pages when swipes is off? Can certain swipes get disabled while keeping others around?

**Comment by ddvk** [comment]
that was for the rm1 where you have buttons

**Comment by raspbounty** [comment]
@ddvk I do not quit get what you mean with your answer. Is there a way to use the rm1 buttons to flip pages?

**Comment by Eeems** [comment]
> @ddvk I do not quit get what you mean with your answer. Is there a way to use the rm1 buttons to flip pages?

Yes, the left/right button on the rM1 will change pages. The middle button returns to home.

**Comment by raspbounty** [comment]
I got it I was just not aware that If I click on the icon in the lower left hand corner and than on the M I can change the modes and that changes the button functionality. It is probably somewhere explained and I overread it.
